skrub/_similarity_encoder.py

```diff
@@ -132,58 +127,8 @@ def ngram_similarity_matrix(
     return np.nan_to_num(out, copy=False)
 
 
-def get_prototype_frequencies(prototypes: np.ndarray) -> np.ndarray:
-    """
-    Computes the frequencies of the values contained in prototypes
-    Reverse sorts the array by the frequency
-    Returns a numpy array of the values without their frequencies
-    """
-    uniques, counts = np.unique(prototypes, return_counts=True)
-    sorted_indexes = np.argsort(counts)[::-1]
-    return uniques[sorted_indexes], counts[sorted_indexes]
-
-
-def get_kmeans_prototypes(
-    X,
-    n_prototypes: int,
-    hashing_dim: int = 128,
-    ngram_range: Tuple[int, int] = (3, 3),
-    sparse: bool = False,
-    sample_weight=None,
-    random_state: Optional[Union[int, RandomState]] = None,
-) -> np.ndarray:
-    """
-    Computes prototypes based on:
-      - dimensionality reduction (via hashing n-grams)
-      - k-means clustering
-      - nearest neighbor
-    """
-    vectorizer = HashingVectorizer(
-        analyzer="char",
-        norm=None,
-        alternate_sign=False,
-        ngram_range=ngram_range,
-        n_features=hashing_dim,
-    )
-    projected = vectorizer.transform(X)
-    if not sparse:
-        projected = projected.toarray()
-    kmeans = KMeans(n_clusters=n_prototypes, random_state=random_state)
-    kmeans.fit(projected, sample_weight=sample_weight)
-    centers = kmeans.cluster_centers_
-    neighbors = NearestNeighbors()
-    neighbors.fit(projected)
-    indexes_prototypes = np.unique(neighbors.kneighbors(centers, 1)[-1])
-    if indexes_prototypes.shape[0] < n_prototypes:
-        warnings.warn(
-            "Final number of unique prototypes is lower than "
-            + "n_prototypes (expected). "
-        )
-    return np.sort(X[indexes_prototypes])
-
-
 class SimilarityEncoder(OneHotEncoder):
-    """Encode string categorical features to a similarity matrix.
+    """Encode string categorical features to a similarity matrix, useful to capture fuzziness across a small number of categories.
```

Comment:
```suggestion
    """Encode string categorical features to a similarity matrix, useful 
    to capture fuzziness across a small number of categories.
```
Sorry, that line was too long.

Reply:
Unfortunately, cutting the line in half will cause problems, I'm for leaving it, even if it's a little long :)